lib/Template/View.pm

```diff
@@ -411,7 +411,7 @@ sub AUTOLOAD {
     $item =~ s/.*:://;
     return if $item eq 'DESTROY';
 
-    if ($item =~ /^[\._]/) {
+    if (substr($item,0,1) eq '.' || substr($item,0,1) eq '_') {
```

Comment:
maybe store the `substr` result to avoid doing it twice...
`if ( my $first = substr($item, 0, 1) `